articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,87 @@
+When it comes to web development for non-techies the first thought that pops into their heads is a deep dive into a dark abyss. Where you magically open your _third eye_ and manage to talk to the computer in a language only the chosen few can understand. However, this is far from reality. Change sides and take it from a techies perspective. The first thoughts that pop up might be "What framework best fits this task?" or "Should I do this from scratch or customize an existing template?". A great number of us look forward to starting a new project. This to us means learning a different concept or re-enforcing an existing one. But what if you decide to build a new webpage from scratch? Do you remember the basics? How are you going to implement your CSS on the webpage? CSS is part of the backbone of any webpage but do you know that it also has its form of styling? Yes we all know about bootstrap and its ease of use but that is a topic for a different day.
```

Comment:
This introduction is a little confusing. Since the article is about styling in CSS, the introduction should prime the reader with something relevant. You could approach this by discussing how CSS has evolved into a full-fledged language, including a variety or preprocessors, and that the recent complexity poses choices in what kind of styling you want to use. Conversely, you could describe HTML as skeleton that leaves a web page bland and colorless, until CSS is added onto it. Either way, the introduction should definitely address: what is CSS (what does it stand for), why even use it and how does it fit in with HTML, since these are articles that assume no prior knowledge and you just jump right into styling after a vague intro.